diff --git a/wrappers/pyrichdem/richdem/__init__.py b/wrappers/pyrichdem/richdem/__init__.py
index 509bde1..3d36bf3 100644
--- a/wrappers/pyrichdem/richdem/__init__.py
+++ b/wrappers/pyrichdem/richdem/__init__.py
@@ -1,7 +1,7 @@
-import pkg_resources
 import datetime
 import copy
 import numpy as np
+from importlib.metadata import version
 
 try:
   import _richdem
@@ -19,7 +19,7 @@ except:
 
 def _RichDEMVersion():
   return "RichDEM (Python {pyver}) (hash={hash}, hashdate={compdate})".format(
-    pyver    = pkg_resources.require("richdem")[0].version,
+    pyver    = version("richdem"),
     hash     = _richdem.rdHash(),
     compdate = _richdem.rdCompileTime()
   )
